examples/bitcoin/src/bitcoin_types.py

```diff
@@ -18,5 +18,5 @@ class ExecuteGetUtxosResult(Variant, total=False):
 
 
 class ExecuteSendTransactionResult(Variant, total=False):
-    ok: None
+    ok: null
```

Comment:
We should discuss this, we might want it to be void/None. We should discuss how to represent the unit type